back/app/models/home_page.rb

```diff
@@ -106,6 +107,14 @@ def sanitize_info_section_multiloc(attribute)
     self[attribute] = @service.linkify_multiloc(self[attribute])
   end
 
+  def set_craftjs_json
+    return if craftjs_json.present?
+
+    craftjs_filepath = Rails.root.join('config/homepage/default_craftjs.json.erb')
+    json_craftjs_str = ERB.new(File.read(craftjs_filepath)).result(binding)
+    self.craftjs_json = ContentBuilder::LayoutImageService.new.swap_data_images(JSON.parse(json_craftjs_str))
```

Comment:
This is because we're specifying an external URL for the header bg image. This needs to be converted to a data code by creating a layout image, which is what the LayoutImageService does.